tools/test.py

```diff
@@ -317,7 +317,8 @@ def HasRun(self, output):
 class ActionsAnnotationProgressIndicator(DotsProgressIndicator):
   def AboutToRun(self, case):
     case.additional_flags = case.additional_flags.copy() if hasattr(case, 'additional_flags') else []
-    case.additional_flags.append('--test-reporter=./tools/github_reporter/index.js')
+    root_path = abspath(join(dirname(__file__), '../')) + os.sep
```

Comment:
I can't remember - do we still need this change in this PR?

Reply:
Unfortunately we do.